metrics/metrics.go

```diff
@@ -525,6 +526,10 @@ func ReshareStateChange(s ReshareState, beaconID string, leader bool) {
 	reshareLeader.WithLabelValues(beaconID).Set(value)
 }
 
-func ErrorSendingPartial(beaconID string) {
-	ErrorSendingPartialCounter.WithLabelValues(beaconID).Add(1)
+func ErrorSendingPartial(beaconID, address string) {
+	ErrorSendingPartialCounter.WithLabelValues(beaconID, address).Set(1)
+}
+
+func SuccessfulPartial(beaconID, address string) {
+	ErrorSendingPartialCounter.WithLabelValues(beaconID, address).Set(0)
```

Comment:
I'm wondering now: instead of setting this to 0 here, maybe we want a separate metric to track successful partial beacon sent?
Since as we discussed if there is let's say 15 errors in a minute and then a successful partial, on the 3 second network we might not notice if we push metrics every minute, e.g.

This can make it easier to track on the grafana side maybe? But then it would grow relatively quickly, but still if we see one node consistently has less successful partials and more errors we might need to investigate.

Reply:
This was the drawback I mentioned vs a count method - we won’t see intra-minute changes; in fact, a node could be down 59secs of every minute, but if it is up for 1 round coinciding with when we fetch prometheus metrics, we’d assume it’s up.

The issue with the previous count was that we needed to somehow get the beacon period and divide error counts by that to work out how many nodes were down, and we lost sight of which nodes were down.